crates/bevy_render/src/render_resource/uniform_buffer.rs

```diff
@@ -8,6 +8,21 @@ use encase::{
 };
 use wgpu::{util::BufferInitDescriptor, BindingResource, BufferBinding, BufferUsages};
 
+/// Stores data to be transferred to the GPU and made accessible to shaders as a uniform buffer.
+///
+/// Uniform buffers are available to shaders on a read-only basis. Uniform buffers are commonly used to make available to shaders
+/// parameters that are constant during shader execution, and are best used for data that is relatively small in size. For
+/// larger data, or data that must be made accessible to shaders on a read-write basis, consider
+/// [`StorageBuffer`](crate::render_resource::StorageBuffer).
```

Comment:
Note that WebGL2 does not support storage buffers. Alternatives for large amounts of data there are vertex/instance buffers, or data textures, depending on what is the most appropriate for the use case.